snapcraft/internal/errors.py

```diff
@@ -169,8 +178,9 @@ def __init__(self, command, app):
 class InvalidContainerRemoteError(SnapcraftError):
 
     fmt = (
-        '{remote!r} is not a valid LXD remote name.\n'
-        "Colons, spaces and slashes can't be used.\n"
+        "Failed to use LXD remote: "
+        "{remote!r} is not a valid name.\n"
+        "Use a LXD remote without colons, spaces and slashes in the name.\n"
```

Comment:
This is good, but “a LXD” should be “an LXD”. (“An” precedes words that _sound like_ they start with a vowel: a European, an HTML document.)

Reply:
It's pronounced "lex-dee" as a matter of fact, so "a LXD remote" is correct.

Reply:
(And it's worth keeping in mind HTML is a tricky example since some English speakers pronounce the "h" as "haitch")

Reply:
@matthewpaulthomas is the `a` or `an` actually necessary?
What about `Use LXD remote name without colons, spaces ans slashes`

Reply:
Sorry, I had no idea about “lex-dee”. In my defence, it’s [no longer in the readme](https://github.com/lxc/lxd/commit/7bcba143993f8956ef47c01e655501d9391fb806#diff-04c6e90faac2675aa89e2176d2eec7d8), and [barely mentioned elsewhere](https://www.google.com/search?q=pronounced+OR+pronunciation+site%3Alinuxcontainers.org). And yes, an article is needed here: the “Failed to ______” lines might be using headline grammar (“Local resident fails to use LXD remote”), but the suggestions at least should use normal grammar (like “slashes in _the_ name”). So, disregard this.